client/src/pages/proposalDetails/ProposalDetails.js

```diff
@@ -70,40 +85,51 @@ function ProposalConditionalRender(isAdmin) {
       const adminUnpressedNoButtonClassName = 'unpressedNoButton adminButton';
       const [adminNoButtonClassName, setAdminNoButtonClassName] = useState(adminUnpressedNoButtonClassName);
 
-      const [adminVote, changeAdminVote] = useState('Unvoted');
+      async function submitVote(voteDecision) {
+        const response = await axios.post('http://localhost:8000/submitVote', {
+          vote: voteDecision,
+          user_id: 4,
+          proposal_id: 1
+        });
+        if (response.data === null) {
+          console.log("Error in submitting vote.");
+        }
```

Comment:
Use the error handling that Warren mentioned with try/catch blocks instead of just checking that response.data === null